### PR TITLE
Improve color suggestions

### PR DESCRIPTION
**Pull request by enzoferey**
Covers https://github.com/excalidraw/excalidraw/issues/262.

Thinking about the comment by @vjeux about using different color palettes for each type of color in the app, I thought it would make great sense to use one same palette with different levels of lightness to reinforce their role.

So I took the `schemeTableau10` color palette from [d3 palettes](https://github.com/d3/d3-scale-chromatic#api-reference) as @pshihn suggested and modified it based on lightness to have light pastel-like colors for the canvas background (so we easily add on top with contrast), saturated and clean colors for the elements background (so they catch eyes as they are the main elements) and dark colors for strokes (so they highlight elements more effectively).

Canvas background:

Screenshot 2020-01-10 at 23 02 47

Element background:

Screenshot 2020-01-10 at 23 03 12

Element stroke:

Screenshot 2020-01-10 at 23 02 56

I have played a bit around, I feel like I can almost select any color on top of any other and it will look pretty decent. That said, I'm definitely open for suggestions or any ideas you might have. Colors are pretty hard and subjective 😄 

Regarding the code, I did it in the more straightforward way. An object with array of colors as values that we select based on a `type` property that is an enum. Let me know if you want to reorganize it in some other way 👍

### CODE REVIEW

**Comment by vjeux** [comment]
Nice, looks awesome. The only thing that's missing is a white background and black stroke. I don't see how to get this back with those colors

**Comment by vjeux** [comment]
Code looks good

**Comment by enzoferey** [comment]
Right, we should probably include white canvas background and black element stroke as defaults. I agree 👍 

What do you mean by " I don't see how to get this back with those colors" ? Can't we add them so they take the first column of the third row ?

**Comment by vjeux** [comment]
My quote was saying that I can't see how to use those colors in the current PR :) Adding them back would solve it!

**Comment by vjeux** [comment]
If you can rebase, then I'll merge it :)

File: src/components/ColorPicker.tsx [hunk 1]
```diff
@@ -58,3 +48,44 @@ export function ColorPicker({
...
+    "#F28E2C",
+    "#E15759",
+    "#76B7B2",
+    "#59A14F",
+    "#EDC949",
+    "#AF7AA1",
+    "#FF9DA7",
+    "#9C755F",
+    "#BAB0AB"
+  ],
```
Comment (vjeux) [inline src/components/ColorPicker.tsx]: Missing white here

**Comment by enzoferey** [comment]
I merged instead as they were conflicts with the new actions architecture and added the white background for elements as well @vjeux :+1:

**Comment by vjeux** [comment]
Thank you!

**Pull request merged**